tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+minversion=2.6.0
+envlist = py26,py27,py36
+skip_missing_interpreters = True
+
+[testenv]
+usedevelop = true
+extras =
+    test
+whitelist_externals =
+    find
+commands=
+    find . -type f -name "*.pyc" -delete
+    find . -type d -name "__pycache__" -delete
+    {envbindir}/coverage erase
+    {envbindir}/coverage run \
+    {envbindir}/pytest --basetemp={envtmpdir} {posargs}
+    {envbindir}/coverage html -d {toxinidir}/htmlcov/{envname}
+
+[testenv:appveyor]
+basepython = {env:PYTHON:}\python.exe
+commands=
+    {envbindir}/coverage run \
+    -m pytest --basetemp={envtmpdir} {posargs}
+    {envbindir}/coverage html -d {toxinidir}/htmlcov/{envname}
```

Comment:
do you want coverage erase for appveyor?

Reply:
no, was left over from copy